tests/integration/kubernetes/setup.sh

```diff
@@ -34,4 +41,5 @@ set_image_path() {
 
 set_runtime_class
 set_hypervisor_path
+set_kernel_path
```

Comment:
Might also need the following kernel params: `systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0`

Reply:
Also has to be handled via annotations.

Reply:
Not need, cgroups v1 is the default in Mariner.